.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,66 +1,67 @@
-name: "\U0001F41E Bug Report"
-description: Create a report to help us improve lemmy-ui
-title: "[Bug]: "
+name: "Bug report"
+description: Create a bug report to help us improve Lemmy-UI!
+title: ""
```

Comment:
It already has a "bug" label, so no need for the prefix. Just adds more text for us poor humans to read.